### PR TITLE
Honor indentation rules

### PR DESCRIPTION
**Pull request by radare**
I think CONTRIBUTING should specify if we should use tabs or spaces (as well as enforce it in vfmt), there are some .v files using spaces and other using tabs.

### CODE REVIEW

**Comment by spytheman** [comment]
Tabs should be used, vfmt will enforce that, but it is a very good idea to add it to CONTRIBUTING.md in the meantime too.

**Comment by spytheman** [comment]
@radare will you change CONTRIBUTING.md in this PR too?

**Comment by radare** [comment]
ok!

**Comment by radare** [comment]
Done

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,38 @@
+## Code Indentation
```
Comment (donnisnoni95) [inline CONTRIBUTING.md]: I think `Indentation` only

**Comment by radare** [comment]
done

**Comment by radare** [comment]
this PR can grow a lot because of all the indentation fixes that im doing by hhand. i would like to have it merged asap, to avoid bigger reviewing issues and vfmt should be fixed too

**Comment by medvednikov** [comment]
Sorry for the delay @radare 

It's just I thought I'd run vfmt on the entire code base today, but I'm constantly being delayed with this...

Let the machine do the work, don't spend time on it :)

The contributing rules can simply be changed to

> use vfmt

**Comment by medvednikov** [comment]
Tests don't pass @radare 

I'll run vfmt today, this time for certain.

**Comment by radare** [comment]
closing in favor to have vfmt ready asap because it hurts my eyes to read the current unindented code

**Comment by medvednikov** [comment]
@radare I've finally run vfmt on the entire code base.

That took a while, but it's ready now and it all works :)

**Pull request closed without merging**